web/themes/joinup/prototype/sass/components/_sidebar-menu.sass

```diff
@@ -5,7 +5,7 @@
 // Component
 // --------------------------------------------------
 .sidebar-menu
-  z-index: 2
+  z-index: 999
```

Comment:
Also does not need so high index...

![overlay](https://user-images.githubusercontent.com/1923476/29456037-1e3a0d1e-8414-11e7-8032-742bb5b27ec2.png)